Website/src/rendering/src/components/Checkout/PanelDeliveryOptions.tsx

```diff
@@ -61,9 +61,9 @@ const PanelDeliveryOptions = (): JSX.Element => {
             type="radio"
             name="deliveryOption"
             id="deliveryOptionShip"
-            value={DELIVERY_TYPES.ship}
+            value={DeliveryTypes.Ship}
```

Comment:
There are also some other files that could be updated, that are currently using 'Ship' as a magic string.